day-2/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+var quotes = ["Just hire a lawyer", "Empower your team", "Raise your prices", "Ignore most of the advice you're given", "Stop trying to hire a unicorn"];
```

Comment:
and separate this across multiple lines. try to keep lines at or around 80 characters. makes it easier to read